nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -40,7 +42,64 @@ def _normalize_base64(b64_text):
     except (ValueError, TypeError):
         return b64_text
 
-class TestExecute(PreprocessorTestsBase):
+
+def merge_dicts(first, second):
```

Comment:
You could make this a *args and then change the code below to be 
```
outcome = {}
for d in args:
    outcome.update(d)
return outcome
```

Reply:
Also since this is a general pattern you might want to put it in `nbconvert/preprocessors/tests/base.py` and import it here.